docs/Using-EthSigner/Azure.md

```diff
@@ -0,0 +1,50 @@
+description: Signing transactions with key stored in Azure Key Vault
+<!--- END of page meta data -->
+
+# Using EthSigner with Azure Key Vault 
+
+EthSigner supports storing the signing key in a [Azure Key Vault](https://azure.microsoft.com/en-au/services/key-vault/). 
+
+!!! caution 
+    EthSigner with Azure Key Vault does not support TLS. If using a remote Azure Key Vault, apply appropriate 
+    security. 
+
+## Storing Private Key in Azure Key Vault 
+
+Create a signing key in the [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/)
+and register EthSigner as an application for the key. The key must be a base 64 encoded private key for 
+ECDSA for curve secp256k1.
+
+Take note of the following to specify when starting EthSigner: 
+
+* Key vault name 
+* Key name 
+* Key version 
+* Client ID 
+* File containing client secret for the client ID 
+
+## Start Pantheon 
```

Comment:
Why is Pantheon being documented in the Azure signing data? I think its important that people think about their port layout, but its not a function of the Azure usage.

Reply:
Fine for now

Reply:
Leaving as explained.  Down the track this will get split into tutorial vs how-to